plugins/pcap/integration/reference/pivot-with-pcap/step_08.ref

```diff
@@ -1,4 +1,4 @@
-{"ts": "2018-10-30T08:59:47.089835", "uid": "CPWqXM3DsYjHkrX69j", "id.orig_h": "192.168.168.100", "id.orig_p": 61257, "id.resp_h": "17.253.55.211", "id.resp_p": 80, "proto": "tcp", "service": "http", "duration": "38.7ms", "orig_bytes": 288, "resp_bytes": 4080, "conn_state": "SF", "local_orig": null, "local_resp": null, "missed_bytes": 0, "history": "ShADadFf", "orig_pkts": 7, "orig_ip_bytes": 664, "resp_pkts": 6, "resp_ip_bytes": 4400, "tunnel_parents": null, "community_id": "1:UY9rhotoYM0Z0HYmZjBZwc/GhcA="}
-{"ts": "2018-10-30T09:09:06.022106", "uid": "ClKkVB2KudjVql07B3", "id.orig_h": "192.168.168.100", "id.orig_p": 61290, "id.resp_h": "17.253.55.204", "id.resp_p": 80, "proto": "tcp", "service": "http", "duration": "41.73ms", "orig_bytes": 291, "resp_bytes": 4102, "conn_state": "SF", "local_orig": null, "local_resp": null, "missed_bytes": 0, "history": "ShADadFf", "orig_pkts": 7, "orig_ip_bytes": 667, "resp_pkts": 6, "resp_ip_bytes": 4422, "tunnel_parents": null, "community_id": "1:+PG4ulcu1wtMyigJpD3ohP1Jj7E="}
-{"ts": "2018-10-30T09:39:45.713592", "uid": "C8lgDb4MIhArhlh0J", "id.orig_h": "192.168.168.100", "id.orig_p": 61364, "id.resp_h": "72.247.178.26", "id.resp_p": 80, "proto": "tcp", "service": "http", "duration": "2.93m", "orig_bytes": 604, "resp_bytes": 768, "conn_state": "SF", "local_orig": null, "local_resp": null, "missed_bytes": 0, "history": "ShADadFf", "orig_pkts": 24, "orig_ip_bytes": 1672, "resp_pkts": 21, "resp_ip_bytes": 1868, "tunnel_parents": null, "community_id": "1:J5wBtNOAIoy1IBWBRFpAeuM/zqg="}
-{"ts": "2018-10-30T09:43:22.519540", "uid": "C6OJP11NPTa6teXh6b", "id.orig_h": "192.168.168.100", "id.orig_p": 61427, "id.resp_h": "72.247.178.26", "id.resp_p": 80, "proto": "tcp", "service": "http", "duration": "1.92m", "orig_bytes": 302, "resp_bytes": 384, "conn_state": "SF", "local_orig": null, "local_resp": null, "missed_bytes": 0, "history": "ShADadFf", "orig_pkts": 17, "orig_ip_bytes": 1066, "resp_pkts": 15, "resp_ip_bytes": 1172, "tunnel_parents": null, "community_id": "1:9a4Fl8C3ce0tZxYivPMIMH6RcWc="}
+{"_path": null, "_write_ts": null, "ts": "2018-10-30T08:59:47.089835", "uid": "CPWqXM3DsYjHkrX69j", "id": {"orig_h": "192.168.168.100", "orig_p": 61257, "resp_h": "17.253.55.211", "resp_p": 80}, "proto": "tcp", "service": "http", "duration": "38.7ms", "orig_bytes": 288, "resp_bytes": 4080, "conn_state": "SF", "local_orig": null, "local_resp": null, "missed_bytes": 0, "history": "ShADadFf", "orig_pkts": 7, "orig_ip_bytes": 664, "resp_pkts": 6, "resp_ip_bytes": 4400, "tunnel_parents": null, "community_id": "1:UY9rhotoYM0Z0HYmZjBZwc/GhcA="}
```

Comment:
`_path` and `_write_ts` should probably be not `null`, right?

Reply:
The fields are not present in the Zeek TSV we ingest, but are present in the matching schema because we also use that for Zeek JSON. That's why the value is always null.

Reply:
I think this discussion is subsumed by the one below, where I proposed `--embed-metadata` to resolve this. This would strip these fields.